tests/acceptance/bootstrap/SettingsContext.php

```diff
@@ -139,6 +139,26 @@ public function theAdministratorHasGivenUserTheRole(string $user, string $role):
 		);
 	}
 
+	/**
+	 * @When the administrator assigns the role :role to user :user using the settings API
```

Comment:
let's specify the name. and try to use `administrator` for system admin
```suggestion
	 * @When user "Admin" assigns the role :role to user :user using the settings API
```